app/commands/decidim/action_delegator/admin/create_delegation.rb

```diff
@@ -21,6 +21,7 @@ def initialize(form, performed_by)
         # Returns nothing.
         def call
           return broadcast(:invalid) if form.invalid? || self_delegate?
+          return broadcast(:max_grants) if max_grants?
```

Comment:
Something like `above_max_grants` would be more explicit IMO, but it's not really a big deal. Just suggesting.